uat/custom-components/src/main/java/com/aws/greengrass/utils/LoggingHelper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.utils;
+
+import ch.qos.logback.classic.Logger;
+
+import java.util.concurrent.TimeUnit;
+
+public final class LoggingHelper {
+    private static double random;
+    public LoggingHelper() {
+    }
+
+    public static void performLogging(Logger logger, int totalLogNumbers, long logWriteFreqSeconds, int logMsgSizeBytes) {
+        for (int i=0; i < totalLogNumbers; i++) {
+            random = Math.random();
+            StringBuilder sb = generateLogMessage(i, logMsgSizeBytes);
```

Comment:
Why not return the logLine directly instead of the builder

Reply:
Following the style from GG.logging, it seems like they prefer the builder to be passed around.